README.md

```diff
@@ -250,20 +267,22 @@ The **summary.tsv** output file generated by `staramr` contains the following co
 * __Genotype__: The AMR genotype of the isolate.
 * __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) for the isolate.
 * __Plasmid__: Plasmid types that were found for the isolate.
+* __Scheme__: The organism that `MLST` has identified.
+* __Sequence Type__: The hashcode that's assigned when combining all allele types
 
 ### Example
 
-| Isolate ID | Genotype                                                  | Predicted Phenotype                                                                                       | Plasmid                             |
-|------------|-----------------------------------------------------------|-----------------------------------------------------------------------------------------------------------| ------------------------------------|
-| SRR1952908 | aadA1, aadA2, blaTEM-57, cmlA1, gyrA (S83Y), sul3, tet(A) | streptomycin, ampicillin, chloramphenicol, ciprofloxacin I/R, nalidixic acid, sulfisoxazole, tetracycline | ColpVC, IncFIB(S), IncFII(S), IncI1 |
-| SRR1952926 | blaTEM-57, gyrA (S83Y), tet(A)                            | ampicillin, ciprofloxacin I/R, nalidixic acid, tetracycline                                               | ColpVC, IncFIB(S), IncFII(S), IncI1 |
+| Isolate ID | Genotype                                                  | Predicted Phenotype                                                                                       | Plasmid                             | Scheme    | Sequence Type |
+|------------|-----------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|-------------------------------------|-----------|---------------|
+| SRR1952908 | aadA1, aadA2, blaTEM-57, cmlA1, gyrA (S83Y), sul3, tet(A) | streptomycin, ampicillin, chloramphenicol, ciprofloxacin I/R, nalidixic acid, sulfisoxazole, tetracycline | ColpVC, IncFIB(S), IncFII(S), IncI1 | senterica | 11            |
+| SRR1952926 | blaTEM-57, gyrA (S83Y), tet(A)                            | ampicillin, ciprofloxacin I/R, nalidixic acid, tetracycline                                               | ColpVC, IncFIB(S), IncFII(S), IncI1 | senterica | 11            |
 
 ## detailed_summary.tsv
 
 The **detailed_summary.tsv** output file generated by `staramr` contains the following columns:
 
 * __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
-* __Gene/Plasmid__: The particular gene detected from ResFinder, PlasmidFinder, and PointFinder.
+* __Data__: The particular gene detected from ResFinder, PlasmidFinder, and PointFinder.
```

Comment:
Could you add at the end of the description **or the sequence type**.

Reply:
Included in my next commit